app/components/support_interface/ucas_match_action_component.html.erb

```diff
@@ -7,7 +7,7 @@
       <p class="govuk-body">
         <%= action_details %>
       </p>
-      <% if @match.action_needed? %>
+      <% if @match.action_needed? && @match.next_action == :ucas_withdrawal_requested %>
```

Comment:
This component is still used in the Support Interface. I think it could upset our Namespaces spec if I tried to use **UCASMatching**::UCASMatchActionComponent in app/views/**support_interface**/ucas_matches/show.html.erb view.

The services code that moved used to be executed in Support Interface by clicking a button but now it's automated and runs daily. So I think it's ok to keep the view component here.

Reply:
Ah yes that makes sense :+1: